scout/server/blueprints/variant/templates/variant/components.html

```diff
@@ -32,6 +32,25 @@
       </tbody>
     </table>
   {% endif %}
+
+  {% if variant.clnsig_onc %}
+    <table id='clinsig_table' class="table table-bordered table-sm">
+      <thead>
+        <th>Oncogenicity</th>
+        <th>Disease name</th>
+        <th>Revstat</th>
+      </thead>
+      <tbody>
+        {% for classif in variant.clnsig_onc %}
+          <tr>
+            <td>{{ classif.value|capitalize }}</td>
+            <td>{{ classif.dn }}</td>
```

Comment:
But consider formatting these: if we end up storing them with _, it would make sense to show them as " " here, add a " " after the comma separator etc! Almost so that one would have a separate lines for them, as some were quite long, and the card isn't that wide, but as you like!

Reply:
Yeah I think I'd like to store them with _, for consistency. I'll fix here and in the parsing!